content/docs/guides/self-hosted/console.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Console
+menu:
+    userguides:
+        parent: self_hosted
+        identifier: self_hosted_console_service
+        weight: 2
+---
+
+The Pulumi Console allows users to view the stacks they have created, see any past activities recorded for those stacks. It also allows you to manage RBAC for your users. 
+
+<!--more-->
+
+In order for the CLI to persist the state of a particular stack, a user must login to the CLI. In order to login to the CLI, you should have created an account using the Console first.
+
+## Dependencies
+
+The Console service depends on the API service. It relies on the API service for authentication callbacks. Depending on the type of authentication mechanism you choose for your users, the services in the Console container may need to communicate with the API container.
+
+## Prerequisites
+
+* Provide a server or virtual machine to install and run the Pulumi components (see Minimum System Requirements below).
+
+You can run this container on the same host that your API container is running or on a different host. If you are running this container on a separate host than the API, be sure to set the appropriate environment variables (more on that below) in order for the Console to be able to reach the API service.
+
+## Minimum System Requirements
+
+| Type | Properties | Notes |
+| ---- | ---------- | ----- |
+| Compute | 1 CPU core w/ 1 GB memory | |
+
+## What's In The Container?
+
+The Console container is a Node 10-based image. A single web server handles delivering UI static assets to your web browser, as well as handling authentication callbacks.
```

Comment:
Here you say "Node 10", but in https://github.com/pulumi/docs/blob/3a7888476e3e59c5ce1fc6d7bfaf1a1dd69d34a1/static/images/docs/guides/self-hosted/on-prem-intranet-config.png and https://github.com/pulumi/docs/blob/3a7888476e3e59c5ce1fc6d7bfaf1a1dd69d34a1/static/images/docs/guides/self-hosted/on-prem-internet-config.png you say "Node 8.9.x". These should be consistent.

Reply:
Good catch. I have to fix those. We upgraded the container image for the Console to use 10.x _after_ I made those diagrams.

Reply:
Done